src/JobCloud.BE.Domain/Models/JobOfferJustJoinIt.cs

```diff
@@ -0,0 +1,20 @@
+﻿using JobCloud.BE.Domain.Enums;
+
+namespace JobCloud.BE.Domain.Models
+{
+    public class JobOfferJustJoinIt
+    {
+        public int Id { get; set; }
```

Comment:
Idea: Making the IDs UUIDs instead of ints

Reply:
Ints are much faster than guids when you compare them. We don't need the security of guids because the user won't have access to any benefits of knowing the id